README.rst

```diff
@@ -94,6 +94,26 @@ By default, GPU support is built if CUDA is found and ``torch.cuda.is_available(
 It's possible to force building GPU support by setting ``FORCE_CUDA=1`` environment variable,
 which is useful when building a docker image.
 
+Building custom wheels
```

Comment:
The info below is very nice, and is only related to building wheels and packaging. So a more appropriate place would probably be `packaging/README.md`.